src/components/Calendar/Calendar.js

```diff
@@ -0,0 +1,91 @@
+import React, { useEffect, useState } from 'react';
+import { StyleSheet, Text, View, SectionList } from 'react-native';
+import TreatEvent from './TreatEvent';
+// import { gql } from 'apollo-boost';
+// import { useQuery } from '@apollo/react-hooks';
+import { mockEvents } from '../../mockData/mockTreat';
+
+export default function Calendar() {
+    //do calcs here to group into a single date object 
+    //dont need to parse float just JSON parse on fetch 
+
+    const calendarEvents = mockEvents.reduce((eventByDay, event) => {
+        let {eventName, eventStartTime, eventEndTime} = event;
+        eventStartTime = new Date(parseFloat(eventStartTime));
+        eventEndTime = new Date(parseFloat(eventEndTime));
+        const duration = Math.floor((eventEndTime.getTime() - eventStartTime.getTime()) / 60000);
```

Comment:
I'd recommend something like moment.js to avoid any potential timezone/hard coded constant/etc issues.

Reply:
Thank you for this suggestion. I will look into implementing this in a future branch.